datahub-web-react/src/app/domain/DomainsList.tsx

```diff
@@ -113,20 +202,11 @@ export const DomainsList = () => {
                         hideRecommendations
                     />
                 </TabToolbar>
-                <DomainsStyledList
-                    bordered
-                    locale={{
-                        emptyText: <Empty description="No Domains!" image={Empty.PRESENTED_IMAGE_SIMPLE} />,
-                    }}
-                    dataSource={domains}
-                    renderItem={(item: any) => (
-                        <DomainListItem
-                            key={item.urn}
-                            domain={item as Domain}
-                            onDelete={() => handleDelete(item.urn)}
-                        />
-                    )}
-                />
+                {domains && domains.length > 0 ? (
+                    <StyledTable columns={allColumns} dataSource={domainData} rowKey="urn" pagination={false} />
+                ) : (
+                    <NoData description="No Domains!" image={Empty.PRESENTED_IMAGE_SIMPLE} />
```

Comment:
The table should handle showing No Data / empty data for us! 

locale={{
   emptyText: <your component> 
}}

Reply:
Nice, thanks for the hint!